### DISCUSSION TITLE
Will circular progress bars be considered in the future?

### DISCUSSION THREAD

**Discussion by jpgarza93** [irihitech/Semi.Avalonia#211] [Ideas]
First, thanks a lot for the amazing library.

I was wondering if there was a plan for adding Circular Progress Bars in the future.

As such:
[Semi Design  - Circular progress bar](https://semi.design/en-US/feedback/progress#Circular%20progress%20bar)

**Comment by rabbitism** [comment]
Yes it is within scope.

**Reply by jpgarza93** [reply]
this is great to hear, thank you

**Comment by rabbitism** [comment]
Progress Ring is implemented in V11. Please have a try. 
![image](https://github.com/irihitech/Semi.Avalonia/assets/14807942/58554ed8-97a2-45a0-a5ef-630831d97b0b)

**Reply by jpgarza93** [reply]
works great
